x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -166,12 +202,17 @@ export const AlertsList: React.FunctionComponent = () => {
   const isRuleTypeEditableInContext = (ruleTypeId: string) =>
     ruleTypeRegistry.has(ruleTypeId) ? !ruleTypeRegistry.get(ruleTypeId).requiresAppContext : false;
 
+  const onPercentileOptionsChange = useCallback((options: EuiSelectableOption[]) => {
```

Comment:
Why are we using `useCallback` here? I'm not super familiar with these newer hooks but this seems unnecessary?